site/src/components/Resources/Resources.tsx

```diff
@@ -184,5 +183,6 @@ const useStyles = makeStyles((theme) => ({
     display: "flex",
     gap: theme.spacing(0.5),
     flexWrap: "wrap",
+    justifyContent: "right",
```

Comment:
I think you should use `flex-end`. I think "right" is not well supported. This should be the reason for Chromatic getting different styles.